src/ripple/protocol/impl/Rate2.cpp

```diff
@@ -34,6 +34,12 @@ as_amount(Rate const& rate)
 
 }  // namespace detail
 
+Rate
+transferFeeAsRate(std::uint16_t fee)
```

Comment:
This is too easily confused with IOU transfer fees. I'd rename this `nftTransferFeeAsRate` or put it in an nft namespace.

Reply:
I'm good with putting this in the `nft` namespace.